docs/knowledgebase/runtime/events.md

```diff
@@ -10,30 +10,48 @@ when those events are emitted.
 
 ## Declaring an Event
 
-Runtime events are created with the `decl_event!` macro.
+Runtime events are created with the `decl_event!` macro (FRAME v1) or the `#[pallet::event]` macro (in FRAME v2).
 
 ```rust
+// In FRAME v1.
 decl_event!(
-	pub enum Event<T> where AccountId = <T as Config>::AccountId {
+	pub enum Event<T: Config> {
 		/// Set a value.
-		ValueSet(u32, AccountId),
+		ValueSet(u32, T::AccountId),
 	}
 );
+
```

Comment:
this is a nit, but i might separate these blocks of code since it would never make sense they are in the same file.

Reply:
I agree - I did it like that for the sake of making each section look more concise. This will get cleaned up once we bump up the docs versioning.